hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/operation/GetOperation.java

```diff
@@ -22,11 +22,10 @@
 import com.hazelcast.replicatedmap.impl.ReplicatedMapService;
 import com.hazelcast.replicatedmap.impl.record.ReplicatedRecord;
 import com.hazelcast.replicatedmap.impl.record.ReplicatedRecordStore;
-import com.hazelcast.spi.ReadonlyOperation;
 
 import java.io.IOException;
 
-public class GetOperation extends AbstractSerializableOperation implements ReadonlyOperation {
+public class GetOperation extends AbstractNamedSerializableOperation {
```

Comment:
Shouldn't this be implementing `ReadonlyOperation`? Same for other reading operations - `EntrySetOperation`, `ContainsValueOperation`, `ContainsKeyOperation`, ...

Reply:
Nope, the issue is, that the replicated map has all the data anyway locally.
So I thought it doesn't make sense, you can stop writes on quorum not met, but why would you stop reads, if the data is there anyway?

Reply:
yes, as agreed, all read operations even if local, are guarded by quorum now.